providers/tests/google/cloud/transfers/test_facebook_ads_to_gcs_system.py

```diff
@@ -26,12 +26,13 @@
 from airflow.exceptions import AirflowException
 from airflow.models import Connection
 from airflow.utils.process_utils import patch_environ
+from providers.src.airflow.providers.google.cloud.example_dags import example_facebook_ads_to_gcs
```

Comment:
```suggestion
from airflow.providers.google.cloud.example_dags import example_facebook_ads_to_gcs
```

(And then imports'll need re-ordering)

Reply:
It might be worth adding `providers.src.` to the forbidden import list for ruff inside `pyproject.toml`

Reply:
leaving for another day, just need to this PR and get onto other ones